tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -507,35 +533,46 @@ protected void CopyToFast(ref MemoryPoolIterator output)
                         }}
                     }}
 
+                    if({header.TestNotTempBit()})
```

Comment:
Yes is faster, measured before and after, significant but not vast; missed content-length accidentally because its now a snowflake 😢